src/tests/routes/controllers/articles.test.js

```diff
@@ -1,22 +1,24 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
+import faker from 'faker';
 import sinon from 'sinon';
+import slug from 'slug';
 
 import app from '../../../server';
 import { transporter } from '../../../config/mail-config';
+
 import { loginCredentials, existingUser } from '../../db/mockdata/userdata';
 import { generateToken } from '../../../utils/helpers.utils';
+
 // configure chai to use expect
 chai.use(chaiHttp);
 const { expect } = chai;
-
 const articleRequestObject = {
-  title: 'the hope of life',
-  description: 'in times of trouble, where do we find hope',
-  body:
-    'when our peace is no more, when friends are no where to be found. we need to do back to our source.',
-  images: 'image1.jpg, image2.png',
-  tags: 'hope, life, source.',
+  title: faker.lorem.sentence(1),
+  description: faker.lorem.paragraphs(1),
+  body: faker.lorem.paragraphs(3),
+  images: ['image1.jpg', 'image2.png'],
```

Comment:
The join table renders storing the images and tags inside the articles table redundant

Reply:
Ok, i  will do something about that.